source/components/GlobalNavigation/components/GlobalNavigationNotifications/CardLink.js

```diff
@@ -0,0 +1,47 @@
+import React, { useCallback, useContext } from 'react';
+import PropTypes from 'prop-types';
+
+import NotificationsContext from '../../utils/NotificationContext';
+
+const CardLink = ({ track, model, children }) => {
+    const { markAsRead, goToDestination } = useContext(NotificationsContext);
+    const { type, isUnread, uri, latestEventUri } = model;
+    const onClick = useCallback((event) => {
+        event.preventDefault();
+
+        track({
+            action: 'click',
+            category: 'on-site-notifications',
+            label: type,
+            value: isUnread ? 1 : 0,
+        });
+
+        if (isUnread) {
+            markAsRead(uri, true)
+                .then(() => {
+                    goToDestination(latestEventUri);
+                });
+        } else {
+            goToDestination(latestEventUri);
+        }
+    }, [markAsRead, track, model, goToDestination]);
+
+    return (
+        <a
+            className="wds-notification-card__outer-body"
+            href={latestEventUri}
+            onClick={onClick}
+        >
+            {children}
+        </a>
+    );
+};
+
+CardLink.propTypes = {
+    children: PropTypes.node.isRequired,
+    // eslint-disable-next-line react/forbid-prop-types
```

Comment:
Where do we define our eslint settings? Maybe we could just remove this rule instead of needing to add exceptions for it?

Reply:
i've disabled that rule in other PR